articles/libraries/lock-android/v1/passwordless-magic-link.md

```diff
@@ -2,14 +2,16 @@
 toc_title: Passwordless with Magic Link
 description: Passwordless with Magic Link with Lock Android
 ---
-# Lock Android: Passwordless with Magic Link
+# Passwordless with Lock Android
```

Comment:
Headers do not match.

Also, some of these docs are quite long but there is no toc.